index.html

```diff
@@ -308,7 +308,7 @@
         <li>Otherwise, return <code>false</code>.
         </li>
       </ol>
-      <div class="issue" title="🐒 Monkey patch">
+      <div class="issue" title="&#128018; Monkey patch">
```

Comment:
@anssiko, this change is unrelated to the feature and the issue being addressed. Do you mind reverting this? We can do it as a separate fix